### DISCUSSION TITLE
Pagination manual number input

### DISCUSSION THREAD

**Discussion by timo95** [gotson/komga#1557] [Ideas]
### Describe your suggested feature

![grafik](https://user-images.githubusercontent.com/24251362/174748494-0c7bd9e0-5c28-4c75-833c-46f2bc48137a.png)

Add the ability to click on the dots and enter any page number manually.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
https://github.com/vuetifyjs/vuetify/issues/22281#issuecomment-3485393788

**Reply by gotson** [reply]
and https://snips.vuetifyjs.com/application-ui/navigation/pagination

**Comment by gotson** [comment]
This is implemented in the new UI
<img width="877" height="254" alt="image" src="https://github.com/user-attachments/assets/d81a0ddd-bf45-47d0-8360-98cf32cf14d5" />

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.26.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.26.0))
